README.md

```diff
@@ -1,2 +1,221 @@
 # Catalog
+
 This is an app to shoe a catalog of my things
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Catlog] <a name="about-project"></a>
+
+> In this project, we have created a console app that will help you to keep a record of different types of things you own: books, music albums, movies, and games. Everything will be based on the UML class diagram presented below. The data will be stored in JSON files but we also prepare a database with tables structure analogical to your program's class structure.
+
+- Create a main.rb file that will serve as your console app entry-point.
+  - Present the user with a list of options to perform.
+  - Let users choose an option.
+  - If needed, ask for parameters for the option.
+  - Have a way to quit the app.
+- The following options should be available:
+  - List all books
+  - List all music albums
+  - List all movies
+  - List of games
+  - List all genres (e.g 'Comedy', 'Thriller')
+  - List all labels (e.g. 'Gift', 'New')
+  - List all authors (e.g. 'Stephen King')
+  - List all sources (e.g. 'From a friend', 'Online shop')
+  - Add a book
+  - Add a music album
+  - Add a movie
+  - Add a game
+
+## Vedio Demo
+
+[Video](./VedioDemo.mp4)
```

Comment:
[REQUIRED] The video links in your README and PR summary are not working. Kindly add a working link to your README.